apps/remote_control/lib/lexical/remote_control/dispatch/handler.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Lexical.RemoteControl.Dispatch.Handler do
```

Comment:
We should add a moduledoc here. It wasn't initially clear to me how this was intended to be used, until I looked at the tests. 

Suggestion:

```elixir
@moduledoc """
Defines a handler that selectively receives events emitted from a remote control node.

## Usage

Define a handler, specifying the events to be handled and implementing `on/2`:

    defmodule MyHandler do
      alias Lexical.RemoteControl.Dispatch.Handler
      alias Lexical.RemoteControl.Api.Messages
      
      import Messages

      use Handler, [project_compiled()]
      
      def on(project_compiled(), _state) do
        ...
        :ok
      end
    end

Register the handler with the dispatcher:

    Lexical.RemoteControl.Dispatcher.add_handler(MyHandler, :my_state)

"""
```

Reply:
Also, we should make this a behaviour or change the moduledoc in Dispatch.